content/blog/create-a-general-purpose-kubeconfig-file-in-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,261 @@
+
+2. Since launching to the HPE Ezmeral Runtime Enterprise is configured through SAML SSO, a session token is fetched and added to the kubeconfig file each time when you launch to the dashboard. With HPE GreenLake for Containers, the session token is configured to expire after an hour. You will be unable to use the downloaded kubeconfig file to access the cluster after token expiration. You have to re-launch to the dashboard and download again the kubeconfig file.
+
+3. The kubeconfig is generated for the HPE Ezmeral Runtime Enterprise. It contains commands to show version, authenticate and refresh of the environment. The standard kubectl tool installed from the official Kubernetes site does not work with this kubeconfig file. You have to download the HPE kubectl plugin, available from the same dashboard, and use it together with the kubectl and the kubeconfig file. 
+
+You need a cleaner solution to have a kubeconfig file that is not tied to a specific user, has the right set of privileges, and works permanently with the standard kubectl tool. This blog post walks you through the process of creating a general-purpose kubeconfig file that allows you to access and deploy applications to the Kubernetes cluster in HPE GreenLake for Containers. The created kubeconfig files can be used by any external scripts, especially in your CI/CD pipeline setup, to work with the Kubernetes cluster.
+
+## Prerequisites
+You need to download the kubectl binary, together with the HPE kubectl plugin and the kubeconfig file, from the launched HPE Ezmeral Runtime Enterprise Dashboard. The downloaded kubectl binary and its plugin need to be set up in your environment. To simplify the setup process, you export the environment variable `KUBECONFIG` and point it to the downloaded kubeconfig file. With those setups in place, you can access the Kubernetes cluster in the HPE GreenLake for Containers.
+
+You have access to permissions that can create and update the following resources in the Kubernetes cluster:
+
+- Kubernetes Service Account(s)
+- Kubernetes Roles & RoleBindings
+
+## Setup Details
+### Create a Kubernetes Service Account
+You use the following yaml manifest file to create a service account in the Kubernetes cluster. Replace the name `cfe-demo-sa` with your service account name.
+
+
+```json
+# serviceaccount.yml
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cfe-demo-sa
+```
+Run the following commands to create the service account and verify that the service account has been created:
+
+```bash
+$ kubectl apply -f serviceaccount.yaml 
+serviceaccount/cfe-demo-sa created
+
+$ kubectl get serviceaccounts cfe-demo-sa 
+NAME          SECRETS   AGE
+cfe-demo-sa   1         24s
+```
+### Create a Role
+After you have the service account, you create a role with a set of permissions which represents the access rights that you want for your scripts. 
```

Comment:
Change "you create a role" to "you can create a role"
Change "which" to "that" and "represents" to "represent"